### PR TITLE
[new release] ledgerwallet and ledgerwallet-tezos (0.1.0)

### PR DESCRIPTION
**Pull request by vbmithr**
Ledger wallet library for OCaml

- Project page: <a href="https://github.com/vbmithr/ocaml-ledger-wallet">https://github.com/vbmithr/ocaml-ledger-wallet</a>
- Documentation: <a href="https://vbmithr.github.io/ocaml-ledger-wallet/doc">https://vbmithr.github.io/ocaml-ledger-wallet/doc</a>

##### CHANGES:

- First public release

### CODE REVIEW

**Comment by vbmithr** [comment]
@pirbo Please check everything is fine for you.
To mergers: Wait for @pirbo confirmation.

**Comment by pirbo** [comment]
Thank you very much @vbmithr . This is all green for me.
The 3 build failures aren't build failure of `ledgerwallet` but of its (external) dependency `conf-hidapi` so they are all orthogonal.

- The failure on fedora is because the package there is called `hidapi-devel`and not `hidapi`.
- I suspect the failure on alpine to be caused by the use of a """too""" old version of alpine : hidapi has only been included in alpine in the very very last v3.11
- I didn't dig into the failure on opensuse.

File: packages/ledgerwallet-tezos/ledgerwallet-tezos.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage: "https://github.com/vbmithr/ocaml-ledger-wallet"
+bug-reports: "https://github.com/vbmithr/ocaml-ledger-wallet/issues"
+dev-repo: "git+https://github.com/vbmithr/ocaml-ledger-wallet"
+doc: "https://vbmithr.github.io/ocaml-ledger-wallet/doc"
+build:    [ "dune" "build"   "-p" name "-j" jobs ]
+# run-test: [ "dune" "runtest" "-p" name "-j" jobs ]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "dune" {>= "2.4.0"}
+  "ledgerwallet"
```
Comment (kit-ty-kate) [inline packages/ledgerwallet-tezos/ledgerwallet-tezos.0.1.0/opam:15]: Would it be ok to sync those two packages and add a `{= version}` constraint here?

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**